core/templates/app.constants.ts

```diff
@@ -230,5 +230,9 @@ export const AppConstants = {
       DISPLAY_NAME: 'Question Reviews'
     }
   },
-  CONTRIBUTOR_BADGE_INITIAL_LEVELS: [1, 10, 20, 50, 100, 200, 300, 400, 500]
+  CONTRIBUTOR_BADGE_INITIAL_LEVELS: [1, 10, 20, 50, 100, 200, 300, 400, 500],
```

Comment:
What is this variable used for? Please add a comment with an explanation.

Reply:
I did not add this variable! I just added a 'comma' to add one more object in this file.